### PR TITLE
seed planted

### PR DESCRIPTION
**Pull request by benjmac**
@zcaceres @abc516 please check out the seed changes

### CODE REVIEW

File: db/seed.js [hunk 1]
```diff
@@ -20,52 +21,265 @@ const users = seed(User, {
...
+    quote: "We must walk consciously only part way toward our goal and then leap in the dark to our success.",
+    image: 'cdn.shopify.com/s/files/1/0273/4903/products/henry-david-thoreau-quote-fridge-magnet-3_large.jpg?v=1380465704',
+    itemNumber: 12,
+    description: 'High-quality button style magnet with full magnetic back.',
+    price: 3.95,
+    size: [2, 3],
+    mood: ['happy']
+  },
+  mag13: {
+    title: 'Oscar Wilde Shaw Fridge Magnet #1',
```
Comment (zcaceres) [inline db/seed.js]: Oscar Wilde Shaw??? ;-)

File: db/seed.js [hunk 2]
```diff
@@ -20,52 +21,265 @@ const users = seed(User, {
...
+    quote: "You can never be overdressed or overeducated.",
+    image: 'cdn.shopify.com/s/files/1/0273/4903/products/oscar-wilde-quote-fridge-magnet-1_large.jpg?v=1380466791',
+    itemNumber: 13,
+    description: 'High-quality button style magnet with full magnetic back.',
+    price: 3.95,
+    size: [2, 3],
+    mood: ['happy']
+  },
+  mag14: {
+    title: 'Oscar Wilde Shaw Fridge Magnet #2',
```
Comment (zcaceres) [inline db/seed.js]: Same

File: db/seed.js [hunk 3]
```diff
@@ -20,52 +21,265 @@ const users = seed(User, {
...
+    quote: "I don't want to go to heaven. None of my friends are there.",
+    image: 'cdn.shopify.com/s/files/1/0273/4903/products/oscar-wilde-quote-fridge-magnet-2_large.jpg?v=1380466908',
+    itemNumber: 14,
+    description: 'High-quality button style magnet with full magnetic back.',
+    price: 3.95,
+    size: [2, 3],
+    mood: ['happy']
+  },
+  mag15: {
+    title: 'Oscar Wilde Shaw Fridge Magnet #3',
```
Comment (zcaceres) [inline db/seed.js]: ?

**Review by zcaceres — COMMENTED** [review]
See inline comments (minor issue) otherwise fine

**Review by abc516 — APPROVED** [review]

**Pull request merged**